dtfs-central-api/src/constants/batch-saving.ts

```diff
@@ -0,0 +1 @@
+export const CHUNK_SIZE_FOR_SQL_BATCH_SAVING = 100;
```

Comment:
If we're making this commonised across central-api I wonder if we need specify what this chunk is saving for. One of the main reasons we do this is SQL parameter query limits which is very dependent on how large the table you are saving to (and how many columns there are). Maybe as there is one we don't need to care about this now but can imagine in catching people out in future